pkg/pulumiyaml/codegen/gen_program_test.go

```diff
@@ -132,9 +132,9 @@ func (m FakePackage) IsComponent(typeName pulumiyaml.ResourceTypeToken) (bool, e
 		// note this returns if err *equals* nil.
 		return false, nil
 	}
-	msg := fmt.Sprintf("Unexpected type token in IsComponent: %q", typeName)
-	m.t.Logf(msg)
-	return false, fmt.Errorf(msg)
+
+	m.t.Logf("Unexpected type token in IsComponent: %q", typeName)
```

Comment:
Is this really better? We've duplicated the text line

Reply:
linters say so, what do you think, should we disable this lint?

Reply:
I think disabling this linter is reasonable.

Reply:
I don't think we should disable this linter. The lint is useful, and we were doing something bad here. I would suggest a different work-around:

```go
		msg := fmt.Sprintf("Unexpected type token in IsComponent: %q", typeName)
		m.t.Logf("%s", msg)
		return "", errors.New(msg)
```